add callout about `getCloudflareContext` in "async mode" (#75)
opennextjs/docs: pages/cloudflare/bindings.mdx

}
```

<Callout type='info'>

> 	`getCloudflareContext` can only be used in SSG routes in "async mode" (where the function's returned value is wrapped
> 	in a promise), to run the function in such a way simply provide an options argument with the `async` flag set to `true`:

Reviewer: ```suggestion
	`getCloudflareContext` can only be used in SSG routes in "async mode" (making it return a promise), to run the function in such a way simply provide an options argument with `async` set to `true`:
```

	```js
	const context = await getCloudflareContext({ async: true });
	```

> 	**WARNING**: During SSG caution is advised since secrets (stored in `.dev.vars` files) and local development

Reviewer: Nice!

We should really have an HowTo to explain the setup...
